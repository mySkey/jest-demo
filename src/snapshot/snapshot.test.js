import { testData } from ".";

// snapshot
test("snapshot", () => {
  // 生成快照文件
  expect(testData()).toMatchSnapshot();
});

// snapshot: inline
test("snapshot: inline", () => {
  expect(testData()).toMatchInlineSnapshot(`
    {
      "age": 28,
      "name": "mySkey",
      "time": "",
    }
  `);
});
