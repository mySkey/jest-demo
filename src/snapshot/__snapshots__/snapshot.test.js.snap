// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
{
  "age": 28,
  "name": "mySkey",
  "time": "",
}
`;
